lib/plugins/aws/deploy/index.js

```diff
@@ -124,7 +125,7 @@ class AwsDeploy {
         log.notice(
           `\n${style.noticeSymbol('✔')} Service deployed to stack ${this.serverless
             .getProvider('aws')
-            .naming.getStackName()}\n`
+            .naming.getStackName()} ${style.aside(`(${getCommandRunTime()}s)`)}\n`
```

Comment:
I think this timer needs to be consistent with one that runs in _progress bar_. 

At least otherwise there'll be a situation where in progress it ends at e.g. `21s` and we're displaying here `27s`, which from user perspective looks like a bug

and for that I think we may setup `pluginManager.commandRunStartTime = Date.now()` (in `pluginManager.run`) and simply resolve afterwards time from that (no need for `getCommandRunTime` util - eventually we may rely on util as [`ms`](https://www.npmjs.com/package/ms) to display then from duration a  human readable duration (still then it'll be nice to also use it for progress display so it's consistent)

Reply:
That's a good catch - I've originally set pretty much the same logic in `pluginManager.run` and then decided to move it to `Serverless` class which was a bad idea - I'll move it back.

Reply:
I've changed it as suggested - I'm currently resolving that each time which might be error prone, I think it would be good to switch to an util that handles that in the future